modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/PaginationBarTag.java

```diff
@@ -333,6 +350,7 @@ protected int processStartTag() throws Exception {
 	private Integer _ellipsisBuffer;
 	private List<PaginationBarDelta> _paginationBarDeltas;
 	private PaginationBarLabels _paginationBarLabels;
+	private boolean _showDropDown = true;
```

Comment:
Do you need the default?

Reply:
I'm actually not sure (but I do think so): I only want to render the DropDown menu, if the `paginationBarDeltas` attribute is not passed and I couldn't think of another way to do it:

If `paginationBarDeltas` is not passed, we assign a default value
and we set `_showDropDown` to `false` in order to not render the DropDown.

If there's a better way to do this, I'm interested.